src/domain/payments/payment-flow.ts

```diff
@@ -82,3 +93,64 @@ export const PaymentFlow = <S extends WalletCurrency, R extends WalletCurrency>(
     recipientWalletDescriptor,
   }
 }
+
+export const PaymentFlowFromLedgerTransaction = <
+  S extends WalletCurrency,
+  R extends WalletCurrency,
+>(
+  ledgerTxn: LedgerTransaction<S>,
```

Comment:
This is in the `payments` module. Should it know what a `LedgerTransaction` is, or should this method be in the app layer?